src/Apps/Artist/Components/ArtistHeader/ArtistHeader2.tsx

```diff
@@ -203,11 +203,17 @@ const ArtistHeader: React.FC<ArtistHeaderProps> = ({ artist }) => {
                     label={insight.label}
                     pb={1}
                   >
-                    <Text variant="sm" color="black60" pb={1}>
-                      {insight.entities.length > 0
-                        ? insight.entities.join(", ")
-                        : insight.description}
-                    </Text>
+                    <Description
+                      variant="sm"
+                      color="black60"
+                      pb={1}
+                      dangerouslySetInnerHTML={{
+                        __html:
+                          insight.entities.length > 0
+                            ? insight.entities.join(", ")
+                            : insight.description ?? "",
```

Comment:
Can we use the `ArtistCareerHighlight` component here as well?

Reply:
I can't see the upload but I'm assuming the answer is: use the `ArtistCareerHighlightFragmentContainer` and the associated fragment.